### DISCUSSION TITLE
Documentation for Previous Version

### DISCUSSION THREAD

**Discussion by acoroza-capital** [primefaces/community#1024] [PrimeReact]
I am working on an old project which is still using react v16.
The primereact compatible to it is version 5.
My problem now is I cannot find the documentation for this version. 
The official documentation has a dropdown for choosing the doc version but only 10(the latest) and v9 are available.
It would be good if there is an easy way to get into older version's documentation because there are lots of devs facing the same problem

**Comment by melloware** [comment]
cc @nitrogenous @mertsincan I know Optimius removed the old showcases that used to be in the list.
